leproxy/plan.sh

```diff
@@ -0,0 +1,19 @@
+pkg_name=leproxy
+pkg_origin=core
+pkg_version="20180113"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("MIT")
+pkg_source="https://github.com/artyom/${pkg_name}/releases/download/${pkg_version}/leproxy-linux-amd64.tar.gz"
+pkg_shasum="c8a7ee698c8f4b4f828905d13978de30a79d6d394f992d8b1a170469c8e0d148"
+pkg_deps=(core/glibc)
```

Comment:
I believe you don't need this at runtime or buildtime.  The binary, `leproxy` is self-contained and not linked to any shared libraries.

```bash
[84][default:/hab/pkgs/bdangit/leproxy/20180113/20180730153850/bin:0]# patchelf --print-interpreter leproxy
cannot find section .interp
[85][default:/hab/pkgs/bdangit/leproxy/20180113/20180730153850/bin:1]# patchelf --print-needed leproxy
cannot find section .dynamic
[86][default:/hab/pkgs/bdangit/leproxy/20180113/20180730153850/bin:1]# patchelf --print-rpath leproxy
cannot find section .dynamic
[87][default:/hab/pkgs/bdangit/leproxy/20180113/20180730153850/bin:1]# file leproxy
leproxy: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, stripped
[88][default:/hab/pkgs/bdangit/leproxy/20180113/20180730153850/bin:0]# ldd leproxy
        not a dynamic executable
```